### PR TITLE
Adding some suggested issue templates for proposals

### PR DESCRIPTION
**Pull request by aarongustafson**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/proposal-without-explainer.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## Alternate Approaches (Optional)
+
+*Could this challenge be addressed another way? Including other options you considered can help others understand the limitations or adverse side effects of other approaches.*
+
+## Privacy & Security Considerations
+
+*Perform a [self-review](https://w3ctag.github.io/security-questionnaire/). You may think of additional privacy/security considerations other than the ideas listed. If you find no considerations after careful review: "No considerable privacy or security concerns are expected, but we welcome community feedback."*
+
+## Open Questions (Optional)
```
Comment (travisleithead) [inline .github/ISSUE_TEMPLATE/proposal-without-explainer.md:41]: I like this section a lot--an encouragement to provide specific feedback. I would title it "Let's Discuss..." :) (but "Open Questions" works too).

**Review by travisleithead — CHANGES_REQUESTED** [review]
Really appreciate this suggestion for some issue templates! Left a few suggestions for improvements for you to consider. Would love to see another reviewer as well. @yoavweiss @cwilso @LJWatson

**Review by yoavweiss — APPROVED** [review]
LGTM % Travis' comments

File: .github/ISSUE_TEMPLATE/proposal-without-explainer.md (hunk 1 above)
Comment (aarongustafson) [reply .github/ISSUE_TEMPLATE/proposal-without-explainer.md]: Done.

File: .github/ISSUE_TEMPLATE/proposal-with-explainer.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+name: New Proposal With an Explainer
+about: If you've got a new idea and created an Explainer elsewhere, use this template
+title: 'Proposal: PROPOSAL NAME'
+labels: with-explainer
```
[the remark this answers is not in the document]
Comment (aarongustafson) [reply .github/ISSUE_TEMPLATE/proposal-with-explainer.md]: Done.

File: .github/ISSUE_TEMPLATE/proposal-with-explainer.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+name: New Proposal With an Explainer
+about: If you've got a new idea and created an Explainer elsewhere, use this template
+title: 'Proposal: PROPOSAL NAME'
```
[the remark this answers is not in the document]
Comment (aarongustafson) [reply .github/ISSUE_TEMPLATE/proposal-with-explainer.md]: Done.

**Comment by aarongustafson** [comment]
All set. Thanks!

**Comment by travisleithead** [comment]
Thanks!

**Pull request merged**